interface/patient_file/deleter.php

```diff
@@ -129,14 +129,22 @@ function delete_drug_sales($patient_id, $encounter_id = 0)
     }
 }
 
-// Delete a form's data from its form-specific table.
+// Delete a form's data that is specific to that form.
 //
-function form_delete($formdir, $formid)
+function form_delete($formdir, $formid, $patient_id, $encounter_id)
 {
     $formdir = ($formdir == 'newpatient') ? 'encounter' : $formdir;
     $formdir = ($formdir == 'newGroupEncounter') ? 'groups_encounter' : $formdir;
     if (substr($formdir, 0, 3) == 'LBF') {
-        row_delete("lbf_data", "form_id = '" . add_escape_custom($formid) . "'");
+        row_delete("lbf_data", "form_id = '$formid'");
```

Comment:
revert above line change. all the variables in the row_delete where parameter need to be escaped with add_escape_custom()